Makefile

```diff
@@ -18,15 +18,20 @@ gen-contracts:
 	cd contracts/eth/nearbridge/ && yarn && yarn build
 	cd contracts/eth/nearprover/ && yarn && yarn build
 	
+# start near blockchain and connect with ganache.
+start-ethash:
+	cli/index.js clean
+	cli/index.js prepare --core-src ${HOME}/Desktop/core
```

Comment:
Maybe we should change this absolute path to relative? Or in some special folder directly in the homedir?